change/@ni-nimble-components-c5cf9aa2-8b3b-430e-99db-7c820f097578.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc minor thing, I try to follow build failures for patterns. Can you merge from main so the build stops failing for the branch?